edu.gemini.lch.model/src/main/java/edu/gemini/lch/model/Interval.scala

```diff
@@ -0,0 +1,28 @@
+package edu.gemini.lch.model
+
+import java.time.Instant
+
+/**
+ * An interval is a period of time between two instants.
+ * Includes start, excludes end.
+ * Requirement: end >= start.
+ * They have fixed ms duration, i.e. end - start and are not comparable.
+ * To compare intervals, you should compare their lengths, i.e. Durations.
```

Comment:
A long time ago Rob wrote a Java `Interval` and there the comparison was based first on start time.  If the same, then the shorter of the two (if any) would come first.

I guess in general this seems a bit under specified but I think though the goal is to only support the functionality we were using in LTTS.  A `contains(i: Instant)` method seems like it would be useful at least and maybe a `length: Duration`?

Reply:
Implemented.